be/src/olap/rowset/segment_v2/segment_writer.cpp

```diff
@@ -36,12 +37,15 @@ const char* k_segment_magic = "D0R1";
 const uint32_t k_segment_magic_length = 4;
 
 SegmentWriter::SegmentWriter(fs::WritableBlock* wblock, uint32_t segment_id,
-                             const TabletSchema* tablet_schema, const SegmentWriterOptions& opts)
-        : _segment_id(segment_id), _tablet_schema(tablet_schema), _opts(opts), _wblock(wblock) {
+                             const TabletSchema* tablet_schema, const SegmentWriterOptions& opts, std::shared_ptr<MemTracker> parent)
+        : _segment_id(segment_id), _tablet_schema(tablet_schema), _opts(opts), _wblock(wblock), _mem_tracker(MemTracker::CreateTracker(
+                -1, "Segment", parent, false)) {
```

Comment:
Class Segment has a MemTracker with the same name "Segment", how to distinguish them?

Reply:
Yes, At first I add the segment id in the memtracker indeedly. But I find the memtracker still hard to distinguish, so i remove it. I will add segment id in the name agein~~~